client/state/data-layer/wpcom/reader/teams/index.js

```diff
@@ -4,20 +4,26 @@
 import { READER_TEAMS_REQUEST, READER_TEAMS_RECEIVE } from 'state/action-types';
 import wpcom from 'lib/wp';
 
-export function handleTeamsRequest( state, action, next ) {
-	wpcom.req.get( '/read/teams', { apiVersion: '1.2' } )
+export function handleTeamsRequest( store, action, next ) {
+	return wpcom.req.get( '/read/teams', { apiVersion: '1.2' } )
```

Comment:
makes sense to me! 
will move from `reader/teams/index.js` --> `read/teams/index.js`